src/ConfigEditor/ConfigEditor.tsx

```diff
@@ -57,6 +66,11 @@ export function ConfigEditor(props: Props) {
     const res: Secret = await getBackendSrv().post(resourcesURL + '/secret', { secretARN: arn });
     return res;
   };
+  const fetchCluster = async (clusterId: string) => {
+    const res: Cluster = await getBackendSrv().post(resourcesURL + '/cluster', { clusterIdentifier: clusterId });
```

Comment:
I tried this and it's curious that if you get a `400` error (e.g. the clusterId does not exists), the error does not appear in the UI but if it's a 404 or a 500 you get it. Since this is optional, I will wrap this in a `try {} catch` so we don't show the error in any case.